google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -1352,6 +1400,61 @@ public void testComposeWithOptions() {
     assertEquals(expectedBlob1, blob);
   }
 
+  @Test
+  public void testWatchAll() {
+    EasyMock.expect(storageRpcMock.watchAll(BUCKET_NAME1, CHANNEL.toProtobuf(), EMPTY_RPC_OPTIONS))
+        .andReturn(CHANNEL.toProtobuf());
+    EasyMock.replay(storageRpcMock);
```

Comment:
I remembered some discussion on using Mockito for all fresh unit test cases, Does that not apply to storage client?

Reply:
> I remembered some discussion on using Mockito for all fresh unit test cases, Does that not apply to storage client?

Yes, this is true. But reviewer can make an exception.
I think we can submit a bunch of Issue on converting existing tests to Mockito.

Reply:
> > I remembered some discussion on using Mockito for all fresh unit test cases, Does that not apply to storage client?
> 
> Yes, this is true. But reviewer can make an exception.
> I think we can submit a bunch of Issue on converting existing tests to Mockito.

but for that we need to take approval from googler's

Reply:
> I think we can submit a bunch of Issue on converting existing tests to Mockito

For this, just create an issue in the respective GitHub repo, Tag the product owner and ask him directly.

Reply:
Converting existing tests is optional but please do use Mockito for all newly written tests. 5+ years ago there was extensive research and discussion inside Google about Java mocking frameworks, and we made a decision that the benefits of switching to Mockito for new tests outweighed the costs of having two mocking frameworks. We also decided not to require porting of older EasyMock tests onto Mockito.

Reply:
@rahulKQl and @dmitry-fa We can add `mockito `test in separate PR. if we are adding it in this PR then we need to remove the `tearDown()` method of `StorageImplTest.class`

```
@After
  public void tearDown() throws Exception {
    EasyMock.verify(rpcFactoryMock, storageRpcMock);
  }
```

Reply:
In that case, please go ahead. Make sure to mention this in public PR.